README.md

```diff
@@ -277,7 +277,7 @@ Environment variables that affect the back-end server's functionality are stored
 | `SWAGGER_UI_PATH`                       | The base path to run the Swagger UI under for serving of API documentation. Defaults to `/docs/api`.                                                                                                                                     |
 | `TZ`                                    | Time-zone to use for the back-end. Required by the Linux OS that runs the back-end, but not used as application configuration.                                                                                                           |
 | `SHOW_TEST_INDICATOR`                   | A boolean flag (set to `0` for `false`, `1` for `true`) to indicate that an environment is intended for testing purposes (prefixes emails subjects and shows a testing variant of the logo in email notifications). Defaults to `false`. |
-| `PROD_LOG_DEBUG`                        | A boolean flag (set to `0` for `false`, `1` for `true`) to enable debug-level logging in production. Defaults to `false`.                                                                                                                |
+| `LOG_DEBUG`                             | A boolean flag (set to `0` for `false`, `1` for `true`) to enable debug-level logging in production. Defaults to `false`.                                                                                                                |
```

Comment:
Might be worthwhile expanding on this to explain how the default value is true for dev and false for prod.